plutus-scb/app/Main.hs

```diff
@@ -250,8 +250,8 @@ runCliCommand ReportActiveContracts = do
     logInfoN "Active Contracts"
     traverse_ (logInfoN . render) =<< Core.activeContracts
     pure $ Right ()
-runCliCommand (UpdateContract uuid endpoint payload) =
-    Core.updateContract uuid endpoint payload
+-- runCliCommand (UpdateContract uuid endpoint payload) =
+--     Core.updateContract uuid endpoint payload
```

Comment:
Is this coming back?

Reply:
I just commented it out because it didn't compile otherwise :D

Reply:
:galaxy-brain: